### PR TITLE
format the code of providers and mongodb folder

### PR DESCRIPTION
**Pull request by saurabhketech**

### CODE REVIEW

File: src/providers/TagProvider.js [hunk 1]
```diff
@@ -4,29 +4,29 @@ import tag from "../models/constant";
...
+        } else {
+            reject("Invalid Type");
+        }
+        validationResult.then(function(result) {
+            if (!result.isEmpty()) {
+                reject(util.inspect(result.array()));
+            } else {
+                body.type = type;
+                resolve(body);
+            }
```
Comment (saurabhketech) [inline src/providers/TagProvider.js]: extra line

File: src/providers/TagProvider.js [hunk 2]
```diff
@@ -4,29 +4,29 @@ import tag from "../models/constant";
...
 
-		});
+        });
```
Comment (saurabhketech) [inline src/providers/TagProvider.js]: extra line

File: src/mongodb/db.js [hunk 3]
```diff
@@ -5,19 +5,19 @@ var CronJob = require("cron").CronJob;
...
-	// create schema
-	var emailSchema = mongoose.Schema({	}, {
-		collection: "emailSave",
-		strict: false,
-	});
-	var email = conn.model("EMAIL", emailSchema);
+    // create schema
+    var emailSchema = mongoose.Schema({}, {
+        collection: "emailStored",
+        strict: false,
```
Comment (arun-kumar-codes) [inline src/mongodb/db.js]: wrong. schema should have defined columns

File: src/mongodb/db.js [hunk 4]
```diff
@@ -5,19 +5,19 @@ var CronJob = require("cron").CronJob;
...
-	new CronJob("*/15 * * * *", function() {
-		inbox.fetch_email(email); // running this function every 15 min
-	}, null, true);
-	
-	return function(req, res, next) {
-		req.email = email;
-		next();
-	};
-};
+    new CronJob("*/15 * * * *", function() {
```
Comment (arun-kumar-codes) [inline src/mongodb/db.js]: make a new file service/cron.js add this in that file

**Pull request merged**